apps/editors/tests/test_views.py

```diff
@@ -2330,6 +2330,24 @@ def test_viewing_queue(self):
         data = json.loads(r.content)
         eq_(data[str(self.addon.id)], self.editor.display_name)
 
+    def test_display_same_files_only_once(self):
+        '''
```

Comment:
Nit: we use double quotes for the docstrings. Also, the docstring text should end with a full stop.